app/lib/email.server.ts

```diff
@@ -79,6 +79,26 @@ async function send(sendCommandInput: SendEmailCommandInput) {
   }
 }
 
+function maybeThrow(e: any, warning: string) {
+  const errorsAllowedInDev = [
+    'ExpiredTokenException',
+    'NotAuthorizedException',
+    'UnrecognizedClientException',
+  ]
+  const { name } = e as SESv2ServiceException
+
+  if (
+    !errorsAllowedInDev.includes(name) ||
+    process.env.NODE_ENV === 'production'
+  ) {
+    throw e
+  } else {
+    console.warn(
+      `SES threw ${name}. This would be an error in production. Since we are in ${process.env.NODE_ENV}, ${warning}.`
+    )
+  }
+}
```

Comment:
This is really similar to `maybeThrow` from `cognito.server.ts`. In a separate PR, would you please refactor `maybeThrow` so that (a) `errorsAllowedInDev` is an argument and (b) the error type is a generic parameter?

Reply:
https://github.com/nasa-gcn/gcn.nasa.gov/pull/2932